buildpack.md

```diff
@@ -967,6 +969,13 @@ The buildpack ID:
 
 If an `order` is specified, then `stacks` MUST NOT be specified.
 
+The buildpack API:
+ - MUST be in form `<major>.<minor>` or `<major>`, where `<major>` is equivalent to `<major>.0`
+ - MUST describe the implemented buildpack API.
+ - SHALL indicate compatibility with a given lifecycle according to the following rules:
+    - When `<major>` is `0`, the buildpack is only compatible with lifecycles implementing that exact buildpack API.
+    - When `<major>` is greater than `0`, the buildpack is only compatible with lifecycles implementing buildpack API `<major>.<minor>`, where `<major>` of the lifecycle equals `<major>` of the buildpack and `<minor>` of the lifecycle is greater than or equal to `<minor>` of the buildpack.
```

Comment:
Does this mean that if you specify `api = 1` that it isn't compatible with `1.1` b/c `api = 1` is equivalent to `api = 1.0`?

Reply:
If the buildpack specifies `api = 1` it IS compatible with lifecycle where `buildpack = 1.1` b/c `1` (the minor of the lifecycle) is greater than or equal to `0` (the implied minor `0` of the buildpack)

Reply:
happy to reword if it's confusing

Reply:
I think that matches what's written. What about in the inverse case? `api = 1.1` would not work on a lifecycle that sets `buildpack = 1.0`, right?

Reply:
Just chatted with @ekcasey and she confirmed the above.